CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Script to clean extractors' tmp files.
 - Script for RabbitMQ error queue cleanup.
 - Ability to use basic html formatting in the welcome message on the home page. [#51](https://github.com/clowder-framework/clowder/issues/51)
+- Ignore the `update` field when posting to `/api/extractors`. [#89](https://github.com/clowder-framework/clowder/issues/89)
```

Comment:
looks like the changelog is old

Reply:
it's going into `extractor-catalog-wireframe`. I just merged that branch in this one.